planner/scan.go

```diff
@@ -252,7 +252,7 @@ func (n *scanNode) simpleExplain() (map[string]any, error) {
 	}
 
 	// Add the collection attributes.
-	simpleExplainMap[collectionNameLabel] = n.col.Name()
+	simpleExplainMap[collectionNameLabel] = n.col.Name().Value()
```

Comment:
> question: Why have you skipped `HasValue()` here before calling `Value()`. If IRRC the value returned is invalid unless `HasValue() == true`

question: Same as above, but here incase the name doesn't exist would suggest perhaps outputting something useful for the user? i.e. `N/A` or something? but I don't have a hard preference here.

Reply:
If we are scanning a collection, it will have a name